src/twisted/web/test/test_flatten.py

```diff
@@ -574,12 +575,36 @@ def test_cancel(self):
         """
         The flattening of a Deferred can be cancelled.
         """
-        d = Deferred()
+        cancel_count = 0
+        cancel_arg = None
+
+        def check_cancel(cancelled):
```

Comment:
```suggestion
        def checkCancel(cancelled):
```
Let's stick to Twisted house style please ;)